src/Microsoft.ML.Data/Evaluators/Metrics/MulticlassClassificationMetrics.cs

```diff
@@ -71,16 +73,22 @@ public sealed class MulticlassClassificationMetrics
         public double MicroAccuracy { get; }
 
         /// <summary>
-        /// If <see cref="TopKPredictionCount"/> is positive, this is the relative number of examples where
-        /// the true label is one of the top-k predicted labels by the predictor.
+        /// Convenience method for "TopKAccuracyForAllK[TopKPredictionCount - 1]". If <see cref="TopKPredictionCount"/> is positive,
+        /// this is the relative number of examples where
+        /// the true label is one of the top K predicted labels by the predictor.
         /// </summary>
-        public double TopKAccuracy { get; }
+        public double TopKAccuracy => TopKAccuracyForAllK?.LastOrDefault() ?? 0;
```

Comment:
Did it previously fall back to `0` or `Double.NaN`?

When is `TopKAccuracyForAllK` expected to be `null`?

Reply:
- I believe that getting the value of an initialized double member yields 0, so it would be 0 and not NaN
- It would be null if TopKPredictionCount was lesser than 0.

I think the modifications @jasallen made are consistent with previous behavior.